FabricExample/android/app/src/main/java/com/fabricexample/MainApplication.kt

```diff
@@ -37,7 +37,7 @@ class MainApplication : Application(), ReactApplication {
     SoLoader.init(this, false)
     if (BuildConfig.IS_NEW_ARCHITECTURE_ENABLED) {
       // If you opted-in for the New Architecture, we load the native entry point for this app.
-      load(bridgelessEnabled = false)
+      load(bridgelessEnabled = true)
```

Comment:
This and similar code on `iOS` are set to `true` by default from 0.74 so no need to switch it here imo.

Reply:
Yes, but with this it will be easier for us to switch between bridgeless - bridgeful for testing.

Reply:
Ok fine